uiowa/uiowa
# Feature: Nav config updates

This might seem like a long list, but most can probably suffice with a review of the config changes.

* Makes the status for main navigation blocks (used in uids_base) ignored in config.
* Resolves #1473: Adds ability for webmaster to administer the external quick links menu in Collegiate, brand.uiowa.edu, and policy.clas.uiowa.edu - I don't think there are any other custom sites currently that should have access to this menu.
* Removed main navigation blocks from policy.clas.uiowa.edu split.
* Re-ordered the items in SiteNow config_ignore to be alphabetical.
* Re-export change to Collegiate's page entity_view_display that removes quotes from a value.

# To test (if you feel the need)
`blt ds --site=policy.clas.uiowa.edu`
* Visit https://claspolicy.local.drupal.uiowa.edu/admin/config/development/configuration/config-split/policy_clas_uiowa_edu/edit and confirm that nav blocks are removed.
* Visit https://claspolicy.local.drupal.uiowa.edu/admin/appearance/settings/uids_base, change the navigation type, and save.
* Run `drush @claspolicy.local cst` and confirm there are no differences (or at least that the differences listed do not include any `block.block.main_navigation__superfish__*` items).
